.github/workflows/main.yml

```diff
@@ -154,6 +192,7 @@ jobs:
   build_ptoken:
     name: Build p-token
     runs-on: ubuntu-latest
+    needs: [format_and_lint_interface, format_and_lint_ptoken]
```

Comment:
I thought were parallelizing everything -- do you want to remove the `needs:` line?

Reply:
I was going to discuss this with you 😅 – I am currently following the pattern of the repo, but we can change all of them to run in parallel.

Reply:
It can be done in future work, feel free to land this either way